detect_secrets/core/scan.py

```diff
@@ -225,18 +225,19 @@ def _scan_for_allowlisted_secrets_in_lines(
     line_numbers, lines = zip(*lines)
     line_content = [line.rstrip() for line in lines]
     for line_number, line in zip(line_numbers, line_content):
+        context = get_code_snippet(line_content, line_number)
         if not is_line_allowlisted(
             filename,
             line,
-            context=get_code_snippet(line_content, line_number),
+            context=context,
```

Comment:
It'd be nice to have all parameters specified similarly to what you did here (e.g. `line=line`)